seedlet/functions.php

```diff
@@ -345,9 +298,6 @@ function seedlet_content_width() {
  * Enqueue scripts and styles.
  */
 function seedlet_scripts() {
-	// Enqueue Google fonts
-	wp_enqueue_style( 'seedlet-fonts', seedlet_fonts_url(), array(), null );
```

Comment:
I think we may actually want to separately enqueue these, so that child themes will be able to de-queue them when they add their own fonts. If we use a standard `@import`, child themes will always have these loaded, even if they don't use them.

Reply:
That's a good point!

[b48e544](https://github.com/Automattic/themes/pull/2251/commits/b48e544d3f0ea495b3ffb371e64f7c6e1656c40f) enqueues the fonts separately for the front end, and enables them to be filterable for the editor.

What do you and @allancole think of this approach?

Reply:
I think it works — my only question is whether or not we need two separate sass files for `fonts` and `fonts-editor`. I can't think of any standard situations where these would be different. Can we just use a single `fonts.scss` file somewhere and import it into both contexts?

Reply:
I'm not sure how to get around having two sass files because the paths for importing the font are different for editor vs front-end, so two different stylesheets are required. I tried to minimize the code duplication on the sass side by using a [mixin](https://github.com/Automattic/themes/blob/b48e544d3f0ea495b3ffb371e64f7c6e1656c40f/seedlet/assets/sass/base/fonts/_mixin.scss). 

I'm open to suggestions, we could maybe do something clever where we pass a variable to `node-sass` at compilation.

Reply:
Ah, yes ok I see what you mean. I think this seems like the best solution for now then. I'll give it a quick test before I do a final review. 👍